src/main/java/uk/gov/hmcts/reform/ccd/documentam/apihelper/Constants.java

```diff
@@ -39,16 +36,19 @@ private Constants() {
     public static final String CASE_DOCUMENT_NOT_FOUND = "Case document not found";
     public static final String CASE_DOCUMENT_ID_INVALID = "Case document Id is not valid";
     public static final String CASE_ID_NOT_VALID = "Case Id is not valid";
-    public static final String CASE_DOCUMENT_HASH_TOKEN_INVALID = "Case document hash-token is not valid for document Id : %s";
+    public static final String CASE_DOCUMENT_HASH_TOKEN_INVALID = "Case document hash-token is not valid for document"
+        + " Id : %s";
     public static final String CASE_TYPE_ID_INVALID = "Case Type Id Id is not valid";
     public static final String JURISDICTION_ID_INVALID = "Jurisdiction Id is not valid";
     public static final String CLASSIFICATION_ID_INVALID = "Jurisdiction Id is not valid";
-    public static final String EXCEPTION_ERROR_ON_DOCUMENT_MESSAGE = "Exception occurred with operation on document id: %s";
+    public static final String EXCEPTION_ERROR_ON_DOCUMENT_MESSAGE = "Exception occurred with operation on document "
+        + "id: %s";
     public static final String EXCEPTION_ERROR_MESSAGE = "Exception occurred with operation";
     public static final String CASE_ID = "caseId";
     public static final String CASE_TYPE_ID = "caseTypeId";
     public static final String JURISDICTION_ID = "jurisdictionId";
-    public static final String SERVICE_PERMISSION_ERROR = "Service doesn't have sufficient permission on requested API {}";
+    public static final String SERVICE_PERMISSION_ERROR = "Service doesn't have sufficient permission on requested "
+        + "API {}";
```

Comment:
Are we mixing String formatting approaches having both `%s` and `{}`?

Reply:
probably something we have to fix that based on usage. Just fixed check-style error (line length?)